cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -453,19 +453,27 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         },
 
         getRequestData: function() {
+            var sectionIds = $('.outline-section').map(function(){return this.id;}).get()
```

Comment:
Nit: I'd say to add a comment here explaining what this line is doing.
Like grab all the sections, map them to their block_ids, and then using `.get` to convert to an Array.